test/fixtures/simple-integration-app/common/models/stores-updating.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please use a singular name in model files - `store-replacing.json`, `store-updating.json`.